audformat/core/define.py

```diff
@@ -26,39 +40,79 @@ class DataType(DefineBase):
 
 
 class Gender(DefineBase):
-    r"""Gender scheme definitions."""
+    r"""Gender scheme definitions.
+
+    Specifies string values
+    representing gender labels
+    that are recommended to use with a scheme
+    that contains gender labels,
+    e.g.
+    ``audformat.Scheme('str', labels=['female', 'male'])``.
+    The exact string values are not part
+    of the :ref:`scheme specifications <data-header:Scheme>`,
+    and can be changed if desired.
+
+    """
     CHILD = 'child'
     FEMALE = 'female'
     MALE = 'male'
     OTHER = 'other'
 
 
 class IndexField(DefineBase):
-    r"""Index fields defined in
-        :ref:`table specifications <data-tables:Tables>`.
+    r"""Index fields of a table.
+
+    Specifies the string values
+    representing column/field names
+    for a filewise
+    and segmented index.
+    The exact string values are part
+    of the :ref:`table specifications <data-tables:Tables>`,
+    and should never be changed by a user.
+
     """
     FILE = 'file'
     START = 'start'
     END = 'end'
 
 
 class IndexType(DefineBase):
-    r"""Index types defined in
-        :ref:`table specifications <data-tables:Tables>`.
+    r"""Index types of a table.
+
+    Specifies the string values
+    representing a filewise or segmented index.
+    Those string values are returned by
+    :attr:`audformat.Table.type`
+    and :func:`audformat.index_type`.
+    The exact string values are not part
+    of the :ref:`table specifications <data-tables:Tables>`,
```

Comment:
I just realized that this is not correct.

The `IndexType` is part of the format specifications, see

![image](https://user-images.githubusercontent.com/173624/205925095-6b9fbc33-e26b-490a-ae2c-ec74a0105b09.png)

I will update accordingly.

Reply:
I fixed this.